helios-services/src/main/java/com/spotify/helios/servicescommon/coordination/DefaultZooKeeperClient.java

```diff
@@ -389,6 +397,51 @@ public Stat stat(final String path) throws KeeperException {
     return transaction(asList(operations));
   }
 
+  @Override
+  public void setAcl(final String path, final List<ACL> aclList) throws KeeperException {
+    assertClusterIdFlagTrue();
+    try {
+      client.setACL().withACL(aclList).forPath(path);
+    } catch (Exception e) {
+      propagateIfInstanceOf(e, KeeperException.class);
+      throw propagate(e);
+    }
+  }
+
+  @Override
+  public List<ACL> getAcl(final String path) throws KeeperException {
+    assertClusterIdFlagTrue();
+    try {
+      return client.getACL().forPath(path);
+    } catch (Exception e) {
+      propagateIfInstanceOf(e, KeeperException.class);
+      throw propagate(e);
+    }
+  }
+
+  @Override
+  public void initializeAclRecursive(final String path, final ACLProvider aclProvider)
```

Comment:
`initializeAclRecursive()` seems rather specific, and only has one caller -- why not keep this code in the `ZooKeeperAclInitializer`?

(Thought I already added a comment about this yesterday but can't find it anywhere today...)